src/registrar/templates/django/admin/includes/detail_table_fieldset.html

```diff
@@ -155,130 +137,99 @@
 
 {% block field_other %}
     {% if field.field.name == "action_needed_reason_email" %}
-    <div>
-        <div id="action-needed-reason-email-placeholder-text" class="margin-top-05 text-faded">
-            -
+
+        <div class="margin-top-05 text-faded field-action_needed_reason_email__placeholder">
+            &ndash;
         </div>
-        <div>
-            <div id="action-needed-reason-email-readonly" class="display-none usa-summary-box_admin padding-top-0 margin-top-0">
-                <div class="flex-container">
-                    <div class="margin-top-05">
-                        <p class="{% if action_needed_email_sent %}display-none{% endif %}" id="action-needed-email-header"><b>Auto-generated email that will be sent to the creator</b></p>
-                        <p class="{% if not action_needed_email_sent %}display-none{% endif %}" id="action-needed-email-header-email-sent">
-                            <svg class="usa-icon text-green" aria-hidden="true" focusable="false" role="img">
-                                <use xlink:href="{%static 'img/sprite.svg'%}#check_circle"></use>
-                            </svg>
-                            <b>Email sent to the creator</b>
+
+        {{ field.field }}
```

Comment:
Curious, why was this switched from id to class?

Reply:
Consistency